woocommerce/woocommerce
# [NOX] Update offline payment gateway UI

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/52392

This PR updates the Offline Payments Settings page to match the Figma design. Previously, the design was a direct copy via SCSS extend from another component. With the base design PR completed, this update includes the following refinements to align with the Figma specifications:

- Added customized header navigation (This also affects the other payments settings pages)
- Removed activity panel and help tabs
- Updated styling and icons
- Disabled animations

Since this PR only focuses on the design, this PR does not include drag-and-drop functionality for the payment methods.
| Before | After |
|--------|-------|
| ![Screenshot 2024-11-07 at 16 31 19](https://github.com/user-attachments/assets/5bf7649a-cdf2-4ea4-b0ef-a587c7a08e10) | ![Screenshot 2024-11-07 at 16 16 17](https://github.com/user-attachments/assets/6591a676-17f0-4d07-90ed-e34059c52126) |
| ![Screenshot 2024-11-07 at 16 31 30](https://github.com/user-attachments/assets/a9f4e27d-93d7-4758-b163-1bf9fcec8de9) | ![Screenshot 2024-11-07 at 16 05 42](https://github.com/user-attachments/assets/6136d8b2-ca0a-4a41-962a-c4a9b5fbbe80) |



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Create a site with `reactify-classic-payments-settings` feature flag enabled
2. Go to `WooCommerce > Settings > Payments`
3. Confirm that the help tabs are hidden and the page title is `WooCommerce Settings`
4. Click on the chevron icon on `Offline payment methods`
5. Confirm that the page matches the Figma design (excluding drag-and-drop UI)

Figma: xO8YdpKfaVoP2d8NvweZ5I-fi-3106_63535


![Offline Methods](https://github.com/user-attachments/assets/28a0e268-86e3-48ac-8b52-caa83675b22a)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Updated Offline Payments Settings layout to match the design

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
